brut.apktool/apktool-lib/src/main/java/brut/androlib/Androlib.java

```diff
@@ -580,14 +580,18 @@ private void copyExistingFiles(ZipFile inputFile, ZipOutputStream outputFile, by
 
             // No need to create directory entries in the final apk
             if (!entry.isDirectory()) {
-                BrutIO.copy(inputFile.getInputStream(entry), outputFile, buffer);
```

Comment:
Maybe we could leave `BrutIO.copy` but adapt it accepting `inputFile` and `outputFile`. Then we can add the below 5-6 lines of the `try-with-resources` and `IOUtils.copy` into `copy` so we can keep this function clean with one liners instead of duplicating the the code in two places.